Resources/doc/reference/installation.rst

```diff
@@ -28,6 +28,17 @@ Enable the Bundle
     php composer.phar require sonata-project/doctrine-orm-admin-bundle  --no-update # optional
     php composer.phar update
 
+.. note::
+    This bundle supports both the ``1.x`` and the ``2.x`` version of the ``FOS/UserBundle``.
+    You need to add the version to your composer.json manually
+
+
+.. code-block:: bash
+
+    php composer.phar require friendsofsymfony/user-bundle:2.0.*@dev  # for the 2.0 version of the FOSUserBundle
```

Comment:
```friendsofsymfony/user-bundle:~2.0@dev``` should be better while keep consistency with the constraint style in the next snippet.